containers/anndata-to-ui/context/requirements-freeze.txt

```diff
@@ -1,79 +1,55 @@
-aiobotocore==2.12.1
-aiohttp==3.9.3
-aioitertools==0.11.0
-aiosignal==1.3.1
-anndata==0.8.0
+altgraph==0.17.2
+anaconda-anon-usage==0.4.4
+anndata==0.10.8
+archspec==0.2.3
+array_api_compat==1.8
 asciitree==0.3.3
-asn1crypto==1.0.1
-async-timeout==4.0.3
-attrs==23.2.0
-botocore==1.34.51
-brotlipy==0.7.0
-certifi==2024.2.2
-cffi==1.15.0
-chardet==3.0.4
+boltons==23.0.0
+Brotli==1.0.9
+certifi==2024.6.2
+cffi==1.16.0
 charset-normalizer==2.0.4
-click==8.1.7
-cloudpickle==3.0.0
-colorama==0.4.4
-conda==4.12.0
-conda-content-trust==0+unknown
-conda-package-handling==1.8.1
-cryptography==36.0.0
-dask==2024.3.1
-distributed==2024.3.1
-fasteners==0.16
-frozenlist==1.4.1
-fsspec==2024.3.1
-h5py==3.8.0
-idna==3.6
-imageio==2.34.0
-importlib_metadata==7.1.0
-Jinja2==3.1.3
-jmespath==1.0.1
-lazy_loader==0.3
-locket==1.0.0
-MarkupSafe==2.1.5
-msgpack==1.0.8
-multidict==6.0.5
+conda==24.5.0
+conda-content-trust==0.2.0
+conda-libmamba-solver==24.1.0
+conda-package-handling==2.3.0
+conda_package_streaming==0.10.0
+cryptography==42.0.5
+distro==1.9.0
+exceptiongroup==1.2.2
+fasteners==0.19
+frozendict==2.4.2
+future==0.18.2
+h5py==3.11.0
+idna==3.7
+jsonpatch==1.33
+jsonpointer==2.1
+libmambapy==1.5.8
+macholib==1.15.2
+menuinst==2.1.1
+mudata==0.2.4
 natsort==8.4.0
-networkx==3.2.1
 numcodecs==0.12.1
-numpy==1.26.4
-ome-zarr==0.8.3
-packaging==23.2
-pandas==2.0.3
-partd==1.4.1
-pillow==10.2.0
-pip==21.2.4
-psutil==5.9.8
-pycosat==0.6.3
+numpy==2.0.1
```

Comment:
We should make sure `vitessce-python` is updated to support `numpy 2` since it's currently constrained not to be: https://github.com/vitessce/vitessce-python/blob/main/pyproject.toml#L35